tests/tests.go

```diff
@@ -234,3 +234,15 @@ func buildImage(o *option.Option, imageName string) {
 	ginkgo.DeferCleanup(os.RemoveAll, buildContext)
 	command.Run(o, "build", "-q", "-t", imageName, buildContext)
 }
+
+func buildCacheShouldNotExist() error {
+	var buildctlCmd []string
+	buildctlCmd = append(buildctlCmd, "buildctl")
+	buildctlOpts, err := option.New(buildctlCmd)
+	if err != nil {
+		return fmt.Errorf("failed to create buildctl command")
+	}
```

Comment:
nit: can we just make this a `ginkgo.Fail(err.Error())` instead of propagating it up to the caller?